apt-filter.js

```diff
@@ -0,0 +1,37 @@
+//Only start playing with this file if you're done with all the basics
+
+const must = (radio) => radio.filter(rb => rb.checked).length
+
+const findRelevantApts = function (address, minPrice, maxPrice, minRooms, maxRooms, immediate, parking) {
+    let relevantApts = apartments.filter(a =>
```

Comment:
instead of writing `a` as your veriable. I would use `apt` then people can understand what you are doing and read your code a lot better